packages/fuel-indexer-plugin/src/lib.rs

```diff
@@ -24,4 +24,6 @@ pub mod prelude {
     pub use super::types::*;
 
     pub use super::utils::*;
+
+    pub use std::collections::HashSet;
```

Comment:
@deekerno 
- Plugin imports now happen usually in `fuel-indexer-plugin/src/wasm.rs` (for wasm) or `fuel-indexer-plugin/src/native.rs` (for native)
- But for this you can put it in the wasm macro module [here](https://github.com/FuelLabs/fuel-indexer/blob/master/packages/fuel-indexer-macros/src/wasm.rs#L39)
  - Or you can move both the `HashMap` and `HashSet` imports to the proper place `fuel-indexer-plugin/src/wasm.rs` (in this case)

Reply:
Ah, I see. Sounds good to me. 👌🏽